src/common/webviewEvents.ts

```diff
@@ -19,6 +19,17 @@ export const webviewEventNames: WebviewEvent[] = [
     'consoleOutput',
 ];
 
+export type FrameToolsEvent = 'sendMessageToBackend' | 'openInNewTab' | 'recordEnumeratedHistogram' |
+'recordPerformanceHistogram' | 'reportError' | 'openInEditor';
+export const FrameToolsEventNames: FrameToolsEvent[] = [
+    'sendMessageToBackend',
+    'openInNewTab',
+    'openInEditor',
+    'recordEnumeratedHistogram',
+    'recordPerformanceHistogram',
+    'reportError',
+];
```

Comment:
Fun TS trick to avoid repetition (though I'm fine leaving this as-is for consistency with other existing types):

```ts
export const FrameToolsEventNames = [
    'sendMessageToBackend',
    'openInNewTab',
    'openInEditor',
    'recordEnumeratedHistogram',
    'recordPerformanceHistogram',
    'reportError',
] as const;
export type FrameToolsEvent = typeof FrameToolsEventNames[number];
```

Reply:
Or do it for all of the files now...